roles/java/tasks/keystores.yml

```diff
@@ -28,6 +28,7 @@
         mode: "0400"
         dest: "{{ download_location }}"
       register: certcp
+      notify: restart-java-services
```

Comment:
I thikn this handler would be better placed in the task "Import certificates to a keystore" & "Trust certificates CA"